Excercise Equipment is now craftable

flowercuco opened the pull request:
## About The Pull Request

Imagine if you will a humble chaplain who wants nothing more than for all of the spiritual folk on the station to get as massive gains as they can, after finding that they can not just make more exercise equipment and that the station does not have any in public places, they go annoy security enough to get into permabrig only to find out that they cant even unwrench the equipment and move it to the church!!! 

NOT ANYMORE!!!

![jS2aBMBa0B](https://user-images.githubusercontent.com/116288367/200889423-f1b6365c-24c4-4f45-8ca4-c96c9085cf27.png)
crafting recipies

![dreamseeker_O4BgBRsFa8](https://user-images.githubusercontent.com/116288367/200889002-8dd7c927-0745-46a9-a4bc-578c7279042a.gif)
demonstrating unwrenching and wrenching equipment

![dreamseeker_hCFQJZdzoS](https://user-images.githubusercontent.com/116288367/200889019-5f4c8399-d539-4d84-8a3f-7735c3ba1f68.gif)
crafting a punching bag and punching it

Now you can craft as much exercise equipment as you want! May everyone on the station get as strong as possible and not just prisoners. 

Also I changed the message that plays when you try to use exercise equipment someone else is using into a balloon alert.
![dreamseeker_PwNesmcR1f](https://user-images.githubusercontent.com/116288367/200890964-4f9fa3ee-ce07-4e6e-815c-a3f4593d06b1.png)

## Why It's Good For The Game

Access to exercise equipment on some maps is limited to static positions and is currently mostly only for prisoners as every map does not have public exercise equipment. Expanding the access means that you can have a Drill Sargent Head of Security or Captain who commands people use these or allows a psychologist to prescribe healthy exercise habits to their patients. 

I think having the potential for exercise equipment on every map is more fun and also if prisoners get their hands on tools they should be allowed to mess with these to annoy security or aid in their escape.

## Changelog
:cl:
add: the punching bag, bench press, and chest press are all able to be crafted and unanchored.
add: crafting recipes for the above
qol: changed a chat message into a balloon alert
qol: adds screentips to equipment (thanks for suggesting i do this mothblocks!)
/:cl:

flowercuco committed:
adds crafting recipies to make excercise equipment
allows for unsecuring, deconstructing, and construction of excercise equipment
changes the "someone else is using this" chat message to a balloon alert

flowercuco committed:
small formatting error

Farquaar commented:
Can't wait to set up a clown-gym in the bar!

Jackal-boop commented:
> Can't wait to set up a clown-gym in the bar!
Bonus points If you bug the bartender for energy drinks

Wallemations commented:
Unsecuring a punching bag and it remaining on the roof is a cursed visual

Mothblocks commented on code/game/objects/structures/gym.dm:
```diff
@@ -20,6 +20,31 @@
+/obj/structure/punching_bag/wrench_act(mob/living/user, obj/item/tool)
+	tool.play_tool_sound(src)
+	if(anchored)
+		balloon_alert(user, ("unsecuring"))
```
```suggestion
		balloon_alert(user, "unsecured")
```

Mothblocks commented on code/game/objects/structures/gym.dm:
```diff
@@ -20,6 +20,31 @@
+		balloon_alert(user, ("unsecuring"))
+		anchored = FALSE
+	else
+		balloon_alert(user, ("securing"))
```
```suggestion
		balloon_alert(user, "secured")
```

Mothblocks commented on code/game/objects/structures/gym.dm:
```diff
@@ -20,6 +20,31 @@
+		balloon_alert(user, ("unsecure first"))
+		return FALSE
+	tool.play_tool_sound(src)
+	balloon_alert(user, ("deconstructing"))
```
```suggestion
	balloon_alert(user, "deconstructing...")
```

... = pending action

Mothblocks commented on code/game/objects/structures/gym.dm:
```diff
@@ -20,6 +20,31 @@
+
+/obj/structure/punching_bag/crowbar_act(mob/living/user, obj/item/tool)
+	if(anchored)
+		balloon_alert(user, ("unsecure first"))
```
```suggestion
		balloon_alert(user, "unsecure first!")
```

! == failed

Mothblocks commented on code/game/objects/structures/gym.dm:
```diff
@@ -47,7 +72,7 @@
 		return
 	if(obj_flags & IN_USE)
-		to_chat(user, span_warning("It's already in use - wait a bit!"))
+		balloon_alert(user, ("Wait your turn!"))
```
```suggestion
		balloon_alert(user, "wait your turn!")
```

Mothblocks commented on code/game/objects/structures/gym.dm:
```diff
@@ -68,6 +93,31 @@
+/obj/structure/weightmachine/wrench_act(mob/living/user, obj/item/tool)
+	tool.play_tool_sound(src)
+	if(anchored)
+		balloon_alert(user, ("unsecuring"))
```
```suggestion
		balloon_alert(user, "unsecured")
```

Mothblocks commented on code/game/objects/structures/gym.dm:
```diff
@@ -68,6 +93,31 @@
+		balloon_alert(user, ("unsecuring"))
+		anchored = FALSE
+	else
+		balloon_alert(user, ("securing"))
```
```suggestion
		balloon_alert(user, "secured")
```

Mothblocks commented on code/game/objects/structures/gym.dm:
```diff
@@ -68,6 +93,31 @@
+
+/obj/structure/weightmachine/crowbar_act(mob/living/user, obj/item/tool)
+	if(anchored)
+		balloon_alert(user, ("unsecure first"))
```
```suggestion
		balloon_alert(user, "unsecure first!")
```

Mothblocks commented on code/game/objects/structures/gym.dm:
```diff
@@ -68,6 +93,31 @@
+		balloon_alert(user, ("unsecure first"))
+		return FALSE
+	tool.play_tool_sound(src)
+	balloon_alert(user, ("deconstructing"))
```
```suggestion
	balloon_alert(user, "deconstructing...")
```

Mothblocks commented on code/game/objects/structures/gym.dm:
```diff
@@ -20,6 +20,31 @@
 		L.add_mood_event("exercise", /datum/mood_event/exercise)
 		L.apply_status_effect(/datum/status_effect/exercised)
 
+/obj/structure/punching_bag/wrench_act(mob/living/user, obj/item/tool)
```
All of these should support `_act_secondary` as well--we want to formalize *de*construction being on right-click, not left-click.

Mothblocks reviewed (changes requested):
You should also add contextual screentips for tools. Let me know if you need help on that

flowercuco committed:
adds screentips!
changes some syntax and formatting to be better!
changes crowbar and wrench acts to be secondary acts with the right mouse button

flowercuco committed:
oops! that \ was important afterall!!!

Mothblocks commented on code/game/objects/structures/gym.dm:
```diff
@@ -20,22 +39,22 @@
 	tool.play_tool_sound(src)
 	if(anchored)
-		balloon_alert(user, ("unsecuring"))
+		balloon_alert(user, "unsecure")
```
unsecured--past tense, same with tha others

flowercuco commented:
thank you for the help! i've added the screentips for barehand and right mouse with the proper tools and changed the wrench stuff and crowbar stuff to right clicking. let me know if i messed up here haha

flowercuco committed:
corrected tenses

Mothblocks reviewed (approved):
Cute pfp btw

flowercuco commented:
thank you very much!

WEWLAD2222 commented:
Hey buddy, I think you got the wrong gym. Cargonia's Crossfit Corner is three doors down.

flowercuco committed:
Merge branch 'master' into Everyone-Is-Swole
